ludwig/features/feature_utils.py

```diff
@@ -100,23 +100,19 @@ def sanitize(name):
 
 
 def compute_feature_hash(feature: dict) -> str:
-    """
-    This function computes a hash for each feature based on the preprocessing dictionary associated with each feature.
-    The input is always the feature dict, however sometimes, this is called from BaseFeature which dumps the feature
-    dict from a ConfigObject and the preprocessing key corresponds to a nested Preprocessing config. This is why it
-    includes the if/else statement.
+    """This function computes a hash for each feature based on the preprocessing dictionary associated with each
+    feature, as well as the feature's type.
+
     Args:
         feature: Feature dictionary
 
     Returns: Feature hash name
-
     """
-    preproc = feature.get(PREPROCESSING, {})
-    if isinstance(preproc, dict):
-        preproc_hash = hash_dict(preproc)
-    else:
-        preproc_hash = hash_dict(preproc.to_dict())
-    return sanitize(feature[NAME]) + "_" + preproc_hash.decode("ascii")
+    feature_data = dict(
```

Comment:
I would add some simple unit tests to check that features with identical preprocessing but different types have different proc cols.

Reply:
Agreed, going to add tests.